app/views/course/statistics/_table.html.slim

```diff
@@ -1,16 +1,28 @@
 table.table.table-hover
   tr
-    th = t('.serial_number')
-    th = CourseUser.human_attribute_name(:name)
-    th = t('.tutor')
-    th = Course::Level.model_name.human
-    th = t('.experience_points')
+    th.text-center
+      = t('.serial_number')
+    th
+      = CourseUser.human_attribute_name(:name)
+    - if @service.group_managers_hash.present?
+      th
+        = t('.tutor')
+    th.text-center
+      = Course::Level.model_name.human
+    th.text-center
+      = t('.experience_points')
 
   - students.each_with_index do |student, index|
    = content_tag_for(:tr, student)
-     td = index + 1
-     td = link_to_user(student)
-     td = @service.tutors_of(student).map(&:name).join(', ')
-     td = student.level_number
-     td = link_to student.experience_points,
-                  course_user_experience_points_records_path(current_course, student)
+     td.text-center
+       = index + 1
```

Comment:
Sorry I didn't catch this earlier - actually in line 15 you can do `- students.each.with_index(1) do |student, index|` and you need to do `= index + 1` here. One of those cool ruby enumerable methods.